### ISSUE TITLE
Anyone idea can help!

### ISSUE THREAD

**Issue by ascdqz** [AmeenAli/XAI_Transformers#5]
Hello, I'm also trying to use this on vit layers. In the paper, it concluded by using .detach() method, which is kind of like ignoring the part and continue? What's the difference of just ignoring the certain layer and using this detach() method?
